aosp_diff/caas/frameworks/av/0001-Not-To-Merge-Enable-AIDL.patch

```diff
@@ -0,0 +1,58 @@
+From 0d845602f14da7abf62a8e97f31957c4de6b2f8c Mon Sep 17 00:00:00 2001
+From: padmashree mandri <padmashree.mandri@intel.com>
+Date: Tue, 7 May 2024 14:25:49 +0000
+Subject: [PATCH] [Not To Merge]Enable AIDL
+
+Signed-off-by: padmashree mandri <padmashree.mandri@intel.com>
+---
+ media/libaudiohal/FactoryHal.cpp | 7 +++++--
+ 1 file changed, 5 insertions(+), 2 deletions(-)
+
+diff --git a/media/libaudiohal/FactoryHal.cpp b/media/libaudiohal/FactoryHal.cpp
+index f88915d7f4..900acce999 100644
+--- a/media/libaudiohal/FactoryHal.cpp
++++ b/media/libaudiohal/FactoryHal.cpp
+@@ -52,12 +52,11 @@ using InterfaceName = std::pair<std::string, std::string>;
+  */
+ static const std::array<AudioHalVersionInfo, 5> sAudioHALVersions = {
+     // TODO: remove this comment to get AIDL
+-    // AudioHalVersionInfo(AudioHalVersionInfo::Type::AIDL, 1, 0),
++    AudioHalVersionInfo(AudioHalVersionInfo::Type::AIDL, 1, 0),
+     AudioHalVersionInfo(AudioHalVersionInfo::Type::HIDL, 7, 1),
+     AudioHalVersionInfo(AudioHalVersionInfo::Type::HIDL, 7, 0),
+     AudioHalVersionInfo(AudioHalVersionInfo::Type::HIDL, 6, 0),
+     AudioHalVersionInfo(AudioHalVersionInfo::Type::HIDL, 5, 0),
+-    AudioHalVersionInfo(AudioHalVersionInfo::Type::HIDL, 4, 0),
```

Comment:
@padmashree9110 , I just saw your patch to enable AIDL. I'm curious why remove version 4, not all version?

Reply:
This patch is not for merge. Once AOSP rebase is done to latest, this change will be there as per my understanding. Since array limit is 5, i removed last element. We can even increase limit to 6. Anyway if latest code after rebase also doesn't have this change, then i will make limit to 6 and then merge.